.github/workflows/optimize-ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         build-args: |
           VERSION=${{ steps.pom-info.outputs.project_version }}
           REVISION=${{ github.event.pull_request.head.sha }}
-        tags: "registry.camunda.cloud/team-optimize/optimize:pr-${{ github.event.pull_request.head.sha }}"
```

Comment:
Hm, looking at this again now I'm wondering if `team-optimize` is for internal use and `optimize-ee` just for actualy releases 🤔 Looks like the uses are similar in other teams as well. Can we double check this, maybe with other monorepo folk or infra?

Reply:
I think you are right, looking at other teams workflows they are using `team-xyz`

Reply:
actually no other team is using `xyz-ee`, but this I believe is because they are all in single jar already

Reply:
I can see the `tram-z` one being used in those `operate/tasklist-ci-build-reusable.yml`